Source/Fuse.Controls.Panels/Backgrounds/TopFrameBackground.uno

```diff
@@ -49,7 +49,13 @@ namespace Fuse.Controls
 
 		protected override float2 GetContentSize(LayoutParams lp)
 		{
-			if defined(iOS || Android)
+			if defined(iOS) {
+				// on iOS, we always treat the bar as though it's 40 points
+				var height = 40 / AppBase.Current.PixelsPerPoint;
```

Comment:
Wouldn't fixing it at 40 mean we always lose that top bit of the screen, even when it's only 20 in size?